tree/10_ncf_internals/modules/templates/jinja2-templating.py

```diff
@@ -24,8 +24,10 @@
 
 import jinja2
 from jinja2 import Environment, FileSystemLoader, StrictUndefined
+import jinja2.loaders
```

Comment:
i'm not expert on jinja2, does is need an extra dependency ?

Reply:
You are right, this import is not necessary. I will remove it.